elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -170,6 +170,8 @@ private static void from18(ChainedTransformationDescriptionBuilder chainedBuilde
         .setDiscard(DiscardAttributeChecker.UNDEFINED, EMIT_EVENTS)
         .addRejectCheck(new RejectAttributeChecker.SimpleRejectAttributeChecker(ModelNode.TRUE), IGNORE_UNAVAILABLE_REALMS)
         .addRejectCheck(RejectAttributeChecker.DEFINED, EMIT_EVENTS);
+        builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.DYNAMIC_CLIENT_SSL_CONTEXTS));
+        builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.DYNAMIC_CLIENT_SSL_CONTEXT));
```

Comment:
I think we can add a follow up commit to not block this PR but @pferraro from something you said elsewhere, are we missing a subsystem model bump?

I think you said a change to any resource at any stability level should trigger a bump so we probably need this bumping to 19 so the transformer should then be when transforming from19

Reply:
> I think we can add a follow up commit to not block this PR but @pferraro from something you said elsewhere, are we missing a subsystem model bump?

Yes - this requires a model version bump.
Modifying an existing transformer for a version that was already released will cause regressions.